en/SUMMARY.md

```diff
@@ -69,9 +69,9 @@
     * [QAV-R 5" Racer (Pixracer)](frames_multicopter/qav_r_5_kiss_esc_racer.md)
     * [Omnicopter](frames_multicopter/omnicopter.md)
   * [Planes](frames_plane/README.md) 
-    * [Fixed Wing Config/Tuning](config_fw/README.md)
-      * [Fixedwing PID Tuning Guide](config_fw/pid_tuning_guide_fixedwing.md)
-      * [Fixedwing Advanced Tuning Guide](config_fw/advanced_tuning_guide_fixedwing.md)
+    * [Fixeding Config/Tuning](config_fw/README.md)
```

Comment:
```suggestion
    * [Fixedwing Config/Tuning](config_fw/README.md)
```

Reply:
yes. Adapted in commit b89161a25c1952cfef7fee42d0b4ca374f9ec1a0